engines/basic_resources/app/controllers/basic_resources/api/v1/memberships_controller.rb

```diff
@@ -22,6 +23,12 @@ def show
         # POST /api/v1/memberships
         #
         def create
+          group = Group.find_by_id(membership_params[:basic_resource_group_id])
```

Comment:
we discussed about this, but frontend already knows about `basic_resource_*` since it comes with the response already and `group_id` means another thing in that resource. We cannot hide it to the frontend.